unix/datetimepicker.c

```diff
@@ -532,6 +541,68 @@ static void dateTimePickerWidget_class_init(dateTimePickerWidgetClass *class)
 {
 	G_OBJECT_CLASS(class)->dispose = dateTimePickerWidget_dispose;
 	G_OBJECT_CLASS(class)->finalize = dateTimePickerWidget_finalize;
+
+	changedSignal = g_signal_new("changed",
+		G_TYPE_FROM_CLASS(class),
+		G_SIGNAL_RUN_LAST,
+		0,
+		NULL, NULL, NULL,
+		G_TYPE_NONE, 0);
```

Comment:
I'd put the last argument on its own line since it denotes the number of arguments. (I wonder if we even need a signal at all...)